packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/face-value/balance-view/index.css

```diff
@@ -0,0 +1,87 @@
+.balance-view-banner {
+  --details-summary-height: 3.75rem; /* 60px */
+  --details-open-height: 12rem;
+  --details-padding-right: calc(var(--boxel-sp) + 3rem);
+
+  position: relative;
+  width: 100%;
+  height: var(--details-summary-height);
+  border: 1px solid var(--boxel-light-400);
+  border-radius: var(--boxel-border-radius);
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.35);
+  overflow: hidden;
+  transition: height var(--boxel-transition);
+}
+
+.balance-view-banner[open] {
+  min-height: var(--details-open-height);
+  overflow: auto;
+}
+
+.balance-view-banner__summary {
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  height: var(--details-summary-height);
+  padding: 0 var(--details-padding-right) 0 var(--boxel-sp);
+}
+
+.balance-view-banner__summary::marker,
+.balance-view-banner__summary::-webkit-details-marker {
+  display: none;
+  content: '';
+}
+
+.balance-view-banner__summary:focus {
+  outline-color: var(--boxel-highlight);
+  outline-offset: -5px;
+}
+
+.balance-view-banner[open] .balance-view-banner__summary {
+  --details-summary-height: var(--boxel-sp-lg);
+}
+
+/* Show/hide marker */
+.balance-view-banner__marker::before {
+  display: block;
+  content: "Show";
+}
+
+.balance-view-banner[open] .balance-view-banner__marker::before {
+  content: "Hide";
+}
+
+.balance-view-banner__marker {
+  position: absolute;
+  right: var(--boxel-sp-sm);
+  top: var(--boxel-sp);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font: var(--boxel-font-sm);
+  letter-spacing: var(--boxel-lsp-sm);
+}
+
+.balance-view-banner__marker-icon {
+  margin-left: var(--boxel-sp-xs);
+  transform: rotate(90deg);
+}
+
+.balance-view-banner[open] .balance-view-banner__marker-icon {
+  transform: rotate(-90deg);
+}
+
+/* Details */
+.balance-view-banner__details {
+  padding: 0 var(--details-padding-right) 0 var(--boxel-sp);
+}
+
+/* Specific to this card */
+.balance-view-banner[open] .balance-view-banner__summary-content {
+  display: none;
+}
```

Comment:
```suggestion
/* Specific to this card */
.balance-view-banner__summary-content {
  height: 100%;
}

.balance-view-banner[open] .balance-view-banner__summary-content {
  display: none;
}
```

Fixes bug in Safari 14.1.1

![image](https://user-images.githubusercontent.com/39579264/123087260-eccbfe80-d456-11eb-815e-7a4f6854ebf1.png)